lib/ReactViews/Custom/Chart/FeatureInfoPanelChart.jsx

```diff
@@ -34,10 +34,7 @@ class FeatureInfoPanelChart extends React.Component {
   @computed
   get chartItem() {
     return this.props.item.chartItems.find(
-      chartItem =>
-        chartItem.type === "line" &&
-        chartItem.name.indexOf("lat") === -1 &&
-        chartItem.name.indexOf("lon") === -1
+      chartItem => chartItem.type === "line"
```

Comment:
Not sure why we were doing this, but this hides charts with names "Relative something" from the chart panel.
Seems to be related to - https://github.com/TerriaJS/terriajs/commit/79efd766296b89af6f2b4b779fe708b31ff77474